liferay-workspace/tests/build.gradle

```diff
@@ -0,0 +1,14 @@
+apply plugin: 'groovy'
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    testCompile 'org.codehaus.groovy:groovy-all:2.4.4'
+    testCompile 'org.spockframework:spock-core:1.0-groovy-2.4'
+	testCompile gradleTestKit()
+}
+
+check.dependsOn ':blade-tests:initBundle'
+check.mustRunAfter ':liferay-workspace/modules:build'
```

Comment:
Is that how we can reference the "root" build? 

can we try

`check.mustRunAfter ':modules:build'`

Also can we make check "dependsOn".  that way in gradle is the "modules" build fails then gradle wont even run this task since this "check" task depends on a good output of the other one.